recipes/symengine/bld.bat

```diff
@@ -0,0 +1,19 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "%CMAKE_GENERATOR%" ^
```

Comment:
> I believe @msarahan updated the quoting of this? I remember discussing this before somewhere.

If this has something to do with `conda-build` (sorry for my Windows ignorance here :grimacing:) remember that we are downgrading `conda-build` to `1.20.1` everywhere but Win64_PY34, where we are using `1.20.0`.

Those downgrades will be removed as soon as we fix https://github.com/SciTools/conda-build-all/issues/41